rotkehlchen/db/history_events.py

```diff
@@ -337,7 +337,12 @@ def _create_history_events_query(
             group_by_event_ids: bool = False,
     ) -> tuple[str, list]:
         """Returns the sql queries and bindings for the history events without pagination."""
-        base_suffix = f'{HISTORY_BASE_ENTRY_FIELDS}, {EVM_EVENT_FIELDS}, {ETH_STAKING_EVENT_FIELDS} {ALL_EVENTS_DATA_JOIN}'  # noqa: E501
+
+        if isinstance(filter_query, EvmEventFilterQuery):
+            base_suffix = f'{HISTORY_BASE_ENTRY_FIELDS}, {EVM_EVENT_FIELDS} {EVM_EVENT_JOIN}'
+        else:
+            base_suffix = f'{HISTORY_BASE_ENTRY_FIELDS}, {EVM_EVENT_FIELDS}, {ETH_STAKING_EVENT_FIELDS} {ALL_EVENTS_DATA_JOIN}'  # noqa: E501
```

Comment:
this I don't believe improves the query a lot but it is less joins and columns in memory so I believe is a positive change